CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ Release report: https://github.com/wazuh/wazuh/issues/15504
 
 ### Fixed
 
+- Fix agentd IT for python3.10 AMI ([#3973](https://github.com/wazuh/wazuh-qa/pull/3973)) \- (Tests)
```

Comment:
This fix belongs to 4.4.2

Reply:
Done fef630091624dca7f5d705a445925f08ea348ded